_field-types/mapping-parameters/term-vector.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: default
+title: Term vector
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 190
+has_children: false
+has_toc: false
+---
+
+# Term vector
+
+The `term_vector` mapping parameter controls whether term-level information is stored for individual text fields during indexing. This information includes details such as term frequency, position, and character offsets, which can be used for advanced features like custom scoring and highlighting.
+
+By default, `term_vector` is disabled. When enabled, term vectors are stored and can be retrieved using the `_termvectors` API.
+
+Enabling `term_vector` increases index size. Use only when you need detailed term-level data.
+{: .important}
+
+## Configuration options
+
+Available configuration options for `term_vector` are the following:
+
+- `"no"` (default): Term vectors are not stored.
+- `"yes"`: Store term frequencies and basic positions.
+- `"with_positions"`: Store term positions.
+- `"with_offsets"`: Store character offsets.
+- `"with_positions_offsets"`: Store both positions and offsets.
+- `"with_positions_payloads"`: Store positions and payloads (if payloads are indexed).
+- `"with_positions_offsets_payloads"`: Store all term vector data.
+
+## Enabling term_vector on a field
+
+The following request creates an index named `articles` with the `content` field configured to store term vectors including positions and offsets:
+
+```json
+PUT /articles
+{
+  "mappings": {
+    "properties": {
+      "content": {
+        "type": "text",
+        "term_vector": "with_positions_offsets"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Indexing a document
+
+Use the following command to index a sample document:
+
+```json
+PUT /articles/_doc/1
+{
+  "content": "OpenSearch is an open-source search and analytics suite."
+}
+```
+{% include copy-curl.html %}
+
+## Retrieving term vectors
+
+Use the following command to retrieve term-level statistics using the `_termvectors` API:
+
+```json
+POST /articles/_termvectors/1
+{
+  "fields": ["content"],
```

Comment:
Is there already a document which contains information around the possible request parameters which can be passed here?

Reply:
No document yet, I will be working on the API next